tests/js/common/aql/aql-view-arangosearch-ddl-rocksdb.js

```diff
@@ -0,0 +1,78 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertTrue, assertFalse, assertEqual */
+
+const jsunity = require('jsunity');
+const tasks = require('@arangodb/tasks');
+
+function testSuite() {
+  const db = require("@arangodb").db;
+  const dbName = 'TestDB';
+  return {
+    setUp: function() {
+      db._useDatabase("_system");
+      try { db._dropDatabase(dbName); } catch (e) {}
+      db._createDatabase(dbName);
+    },
+    tearDown: function() {
+      db._useDatabase("_system");
```

Comment:
the teardown should unregister the task `CreateLinkInBackgroundMode_AuxTask` (will throw if the task isn't there anymore, so it needs to be wrapped in a try...catch).

Reply:
Added unregistering.